modules/example-browser/src/caching_materials_manager.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* This is a simple example of using a KMS Keyring
+ * to encrypt and decrypt using the AWS Encryption SDK for Javascript in a browser.
+ */
+
+import {
+  KmsKeyringBrowser,
+  KMS,
+  getClient,
+  encrypt,
+  decrypt,
+  WebCryptoCachingMaterialsManager,
+  getLocalCryptographicMaterialsCache
+} from '@aws-crypto/client-browser'
+import { toBase64 } from '@aws-sdk/util-base64-browser'
+
+/* This is injected by webpack.
+ * The webpack.DefinePlugin will replace the values when bundling.
+ * The credential values are pulled from @aws-sdk/credential-provider-node
+ * Use any method you like to get credentials into the browser.
+ * See kms.webpack.config
+ */
+declare const credentials: {accessKeyId: string, secretAccessKey:string, sessionToken:string }
```

Comment:
Do you need to add the sessionToken to all browser examples, or just this one?

Reply:
All of them.